src/content/cloud/okteto-cli.mdx

```diff
@@ -76,6 +76,12 @@ In order to refer to images in the `build` section of the Okteto Manifest, you c
 - `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)
 - `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference (using its SHA)
 
+It is also possible to handle timeouts from the client side when communicating with the buldkit daemon. To do so, the following environment variables can be modified:
+
+- `OKTETO_KEEPALIVE_CLIENT_TIME_MS`: After a duration of this time if the client doesn't see any activity it pings the server to see if the transport is still alive. If set below 10s, a minimum value of 10s will be used instead. The current default value is infinity.
```

Comment:
```suggestion
- `OKTETO_KEEPALIVE_CLIENT_TIME_MS`: After a duration of this time, if the client doesn't see any activity, it will ping the server to see if the transport is still alive. If set below 10s, a minimum value of 10s will be used.The current default value is infinity.
```